src/main/java/de/retest/recheck/review/GlobalChangeSetApplier.java

```diff
@@ -25,7 +25,7 @@
 
 public class GlobalChangeSetApplier {
 
-	private static final ShouldIgnore SHOULD_IGNORE_NOTHING = null;
+	private static final Filter SHOULD_IGNORE_NOTHING = null;
```

Comment:
Did we both overlook it, or should the variable not change? 
Would say there was a reason why I did not rename it, but could be wrong.
CC @beatngu13

Reply:
SHOULD_FILTER_NOTHING